about.html

```diff
@@ -0,0 +1,181 @@
+                  </div>
+                  <div class="row pt-2">
+                    <div class="col-12">
+                      <p class="w-75 d-block m-auto text-center">The last two Javascript Developer Conferences took place in Amsterdam and Norway</p>
+                    </div>
+                    <div class="col-12 py-5">
+                      <div class="row">
+                        <div class="col-md-6 col-12 mb-3">
+                          <div class="year1 position-relative">
+                            <div class="overlay">
+                              <span class="year">2018</span>
+                              <span class="location  text-center">Javascript Developer Conference In Amsterdam</span>
+                            </div>
+                          </div>
+                        </div>
+                        <div class="col-md-6 col-12 mb-3">
+                          <div class="year2 position-relative">
+                            <div class="overlay">
+                              <span class="year">2021</span>
+                              <span class="location text-center">JavaScript Developer Conference In Norway</span>
+                            </div>
+                          </div>
+                        </div>
+                      </div>
+                    </div>
+                  </div>
+                </div>
+              </section>
+              <section class="partners text-white py-5 ">
+                <div class="container py-md-5 py-3">
+                  <div class="row">
+                    <h3 class="section-title light">Partners</h3>
+                  </div>
+                  <div class="row pt-3 pt-md-5">
+                    <div class="partner-lists d-flex d-md-block flex-column align-items-center">
+                      <div class="partner-logo py-5 my-3">
+                        <img src="./img/companies/uber.png" alt="uber">
+                      </div>
+                      <div class="partner-logo py-5 my-3">
+                        <img src="./img/companies/google.png" alt="google">
+                      </div>
+                      <div class="partner-logo py-5 my-3">
+                        <img src="./img/companies/netflix.png" alt="netflix">
+                      </div>
+                      <div class="partner-logo py-5 my-3">
+                        <img src="./img/companies/amazon.png" alt="amazon">
+                      </div>
+                      <div class="partner-logo py-5 my-3">
+                        <img src="./img/companies/meta.png" alt="meta">
+                      </div>
+                      <div class="partner-logo py-5 my-3">
+                        <img src="./img/companies/microsoft.png" alt="microsoft">
+                      </div>
+                    </div>
+                  </div>
+                </div>
+              </section>
+        </main>
+        <footer class=" d-md-block">
```

Comment:
Please be sure to change your background color of the `footer` to a dark tone so that we meet the project requirements 💪🤓

![image](https://user-images.githubusercontent.com/15807343/175354165-abc74d4b-a77b-4248-95fa-ff0b7ec25317.png)